rpc/wrtc_signaling_answerer.go

```diff
@@ -38,17 +38,18 @@ type webrtcSignalingAnswerer struct {
 	// bgWorkersMu should be `Lock`ed in `Stop` to `Wait` on ongoing background workers in startAnswerer/answer.
 	// bgWorkersMu should be `RLock`ed when starting a new background worker (allow concurrent background workers) to
 	// `Add` to bgWorkers.
-	bgWorkersMu     sync.RWMutex
-	bgWorkers       sync.WaitGroup
-	cancelBgWorkers func()
+	bgWorkersMu     sync.RWMutex   // TODO(Bashar): remove after removing deadlock
+	bgWorkers       sync.WaitGroup // TODO(Bashar): remove after removing deadlock
+	cancelBgWorkers func()         // TODO(Bashar): remove after removing deadlock
+	sw              *utils.StoppableWorkers
```

Comment:
Let's still call this `bgWorkers`. FWIW, in golang, we generally use short variables names like `sw` only when that variable is local/has a very limited scope over which it is not difficult to interpret what it does. When it's a field on a struct, we can afford to encode a bit more information in the name.